src/components/agency/AgencyChart.js

```diff
@@ -98,14 +100,12 @@ class AgencyChart extends React.Component {
         <AgencyChartDetails
           colors={colorMap}
           crime={crime}
-          f
           data={active}
           dataPrior={activePriorYear}
           keys={keys}
           noun={noun}
-          since={since}
           updateYear={this.updateYear}
-          until={until}
```

Comment:
I'm not sure I agree with this, here are some of my reasons why:

1. One thing that is nice about props is if they are somewhat consistent from component to component. This isn't always achievable but when we can pass around our usual `since`, `until`, `crime`, etc I think we should because it reduces the cognitive overhead of understanding what that component's interface is. `until` is always an integer and it represents the end of the year range.
2. I know we don't always adhere to it, but I believe it is best to pass as simple an object as possible via the props. This view is partially supported by our linter which complains if we say that a prop is just an array or an object. It wants to know more about those because that's not a very well defined API boundary. That object could have a different shape or the array could have different types of children.

In the end, this is a stylistic decision so I'll defer to you @jpwentz as the soon to be owner of the code base but these are my thoughts.

Reply:
In short you are saying the component should have these important variable even if they do not utilize them?

Reply:
I guess I think that the component should accept `until` and just calculate it's own range inside of the component. This makes the API boundary between the components a bit more predictable IMHO

Reply:
To be clear, if a component doesn't need a prop, we shouldn't pass it in though.

Reply:
OK -- I see your point and great input. I will make the change - just wanted to be clear that I understood.